tests/test_algo.py

```diff
@@ -1,16 +1,16 @@
 # Builtins
+from re import S
 import unittest
 from unittest.mock import patch
 
-from harvest import algo
+#from harvest import algo
```

Comment:
Lets remove this if we don't need it. I find that dead code becomes a problem down the road.